examples/test/BasicComponents_test.js

```diff
@@ -80,3 +80,10 @@ test("Headline should render", async t => {
 test("Heatmap should render", async t => {
     await checkRenderChart(".s-heat-map", t);
 });
+
+test("should render Category legend", async t => {
```

Comment:
it should be `GeoPushpinChart should render `

Reply:
fixed